### DISCUSSION TITLE
How to select text in DataTable

### DISCUSSION THREAD

**Discussion by maitrungduc1410** [primefaces/community#2873] [PrimeReact] [answered]
just now I upgraded to latest version (10.8.4), and notice that I'm no longer be able to select text in DataTable, it's all "draggable" which leads to poor UX. Is there anyway I can turn of this behaviour, I'm checking official PrimeReact docs, but examples there are all draggable

https://github.com/user-attachments/assets/f4385b80-edd3-41f6-add2-478ce2cb05fb

**Comment by maitrungduc1410** [comment]
@melloware can you please have a look?🙏

**Reply by melloware** [reply, marked as answer]
@maitrungduc1410 ticket already opened: https://github.com/primefaces/primereact/issues/7332
